com.flagodna.FastiShare.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2025 Ebbe Zeinstra -->
+<component type="desktop-application">
+
+  <id>com.flagodna.FastiShare</id>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+
+  <name>FastiShare</name>
+  <summary>Fast. file-sharing tool that lets you transfer files between devices on the same local network</summary>
+
+  <developer id="com.flagodna">
+    <name>FlagoDNA Developer</name>
+  </developer>
+
+  <description> 
+    <p>FastiShare is a lightweight, privacy-focused file-sharing tool that lets you transfer files between devices on the same local network without requiring internet access or cloud storage.</p>
+  </description>
+
+  <launchable type="desktop-id">com.flagodna.FastiShare.desktop</launchable>
+
+  <branding>
+    <color type="primary" scheme_preference="light">#4361EE</color>
+    <color type="primary" scheme_preference="dark">#223382</color>
+  </branding>
+
+  <content_rating type="oars-1.1" />
+
+  <url type="bugtracker">https://github.com/Flagodna-Developer/FastiShare/issues</url>
+  <url type="homepage">https://flagodna-developer.github.io/FastiShare</url>
+  <url type="contact">https://github.com/Flagodna-Developer/FastiShare/discussions</url>
+  <url type="faq">https://quelea-projection.github.io/docs/FAQ_(Frequently_Asked_Questions)</url>
+  <url type="translate">https://github.com/Flagodna-Developer/FastiShare/tree/main/translation</url>
+  <url type="contribute">https://github.com/Flagodna-Developer/FastiShare/blob/main/CONTRIBUTING.md</url>
+  <url type="vcs-browser">https://github.com/Flagodna-Developer/FastiShare</url>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://flagodna-developer.github.io/FastiShare/assets/fastishare-preview.png</image>
+      <caption>FastiShare dashboard view</caption>
+    </screenshot>
+  </screenshots>
+
+  <recommends>
+    <control>keyboard</control>
+    <control>pointing</control>
+    <display_length compare="ge">768</display_length>
+  </recommends>
+
+    <releases>
+    <release version="1.0.2" date="2025-05-04">
+      <url type="details">https://github.com/Flagodna-Developer/FastiShare/raw/refs/heads/main/changelog/linux/1.0.2.txt</url>
```

Comment:
this should link to the 1.0.2 tag. not to somme file on the main branch (linking to a branch will break)